app/views/layouts/application.html.erb

```diff
@@ -2,8 +2,8 @@
 <html>
   <head>
     <title>Hydranorth</title>
-    <%= stylesheet_link_tag    "application", media: "all", "data-turbolinks-track" => true %>
```

Comment:
Don't think this application layout is ever used, but cleaning up the data-attrs for turbolinks.